### ISSUE TITLE
SDVs

### ISSUE THREAD

**Issue by MagnetAss** [OFPS-DEV2/becti_zerty_OFPS_guerrilla.Altis#7]
$400 = 2 AR equiped troops /w explosive satchels.

**Comment by SpanishSurfer** [comment]
Explain a bit more.

**Comment by MagnetAss** [comment]
Last night I was using our navy fac as a barracks to get enough armed Ai to take big towns alone. Just buy 4 crewed SDVs and bam, $1,600 for 8 soldiers with the UW rifle and explosive satchels in their packs at gear 0/no barracks. So far I'm the only one to figure it out despite explaining to Live why we needed a naval fac with wide open spaces.

**Comment by SpanishSurfer** [comment]
So the crew came with sdv?

**Comment by MagnetAss** [comment]
Yep. Was able to loot all items for personal inventory as well. Love the CSAT diving mask, I'm not gonna lie.
